### ISSUE TITLE
Using Different Hover Effect For Sign Up Button

### ISSUE THREAD

**Issue by sahajoydeep** [Rajan-Barnwal/jobproject#74]
When Hovered Over the Signup Button, the button text is nearly white.

I can fix this, I would love to fix this issue, Joydeep saha, from GSOC '24

<img width="1440" alt="Screenshot 2024-05-11 at 1 12 40 AM" src="https://github.com/Rajan-Barnwal/jobproject/assets/88198830/1d4bcf5f-a3a7-4de5-af06-0b471e23f8e1">

**Comment by GAGGZ1** [comment]
hey @Rajan-Barnwal 
I suggest applying a unique hover effect to the Sign Up button for an engaging user experience. With my skills in front-end development, I can create an effect that enhances interaction. Let's make the Sign Up process stand out!
could you please assign me

**Comment by sahajoydeep** [comment]
@Rajan-Barnwal Yes, I can also use the same and add further animations and related stuff using the CSS on the signup hover button, I would be glad if I can be assigned.
